opendevin/core/config.py

```diff
@@ -366,8 +417,18 @@ def load_from_toml(config: AppConfig, toml_file: str = 'config.toml'):
         if 'agent' in toml_config:
             agent_config = AgentConfig(**toml_config['agent'])
 
+        # set sandbox config from the toml file
+        sandbox_config = config.sandbox
+        if 'sandbox' in toml_config:
+            sandbox_config = SandboxConfig(**toml_config['sandbox'])
```

Comment:
Just to clarify, this should work great with the new config, but I don't think it works with the existing toml. It would not set the attributes on the SandboxConfig instance.

This is where a test doing just that, attempting to work with an existing config file, with these vars, would show us when it works.

Reply:
Added test